docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -257,12 +278,87 @@ remove_action( 'bigcommerce/template/product/archive', bigcommerce()->templates-
+				'shipping_calc_error'      => __( 'There was an error calculating your shipping cost. Please try again.', 'bigcommerce' ),
+			],
+			'account'    => [
+				'confirm_delete_message' => __( 'Are you sure you want to delete this address?', 'bigcommerce' ),
+				'confirm_delete_address' => __( 'Confirm', 'bigcommerce' ),
+				'cancel_delete_address'  => __( 'Cancel', 'bigcommerce' ),
+			],
+			'errors'     => [
+				'pagination_error'         => __( 'There was an error processing your request. Please try again.', 'bigcommerce' ),
+				'pagination_timeout_error' => __( 'The server took too long to complete this request. Please try again.', 'bigcommerce' ),
+			],
+			'pricing'    => [
+				'loading_prices' => __( 'Retrieving current pricing data...', 'bigcommerce' ),
+			],
+			'inventory'    => [
+ 				'in_stock' => __( 'in Stock', 'bigcommerce' ),
+ 				'out_of_stock' => __( 'Out of Stock', 'bigcommerce' ),
+			],
+			'wish_lists' => [
+				'copied' => __( 'Copied!', 'bigcommerce' ),
+				'copy_link' => __( 'Copy link', 'bigcommerce' ),
+				'copy_success' => __( 'Wish List URL copied to clipboard.', 'bigcommerce' ),
+			],
+		];
+```
+
+As an example of how to localize strings, let's change the message displayed to a user when they add an item to their cart. By default, the message "Product successfully added to your cart." is displayed. Instead, let's update that language and include a link to your store's cart.
+
+1. Using either a child theme or child plugin, declare a function named `update_add_to_cart_message()` that accepts `$js_i18n_array` as an arguement.
+2. Within `update_add_to_cart_message()`, set the new value for the `['ajax_add_to_cart_success']` key located in the nested `['cart']` array to `'Item added to cart. <a href="/cart">View Cart!</a>'`. Note that this is an example and your store's cart might have a different slug than `/cart`.
+3. Return the filtered `$js_i18n_array` array using a return statement.
+
+Your `update_add_to_cart_message()` function will look like the following:
+
+```php
+function update_add_to_cart_message( $js_i18n_array ){
+   $js_i18n_array['cart']['ajax_add_to_cart_success'] = 'Item added to cart. <a href="/cart">View Cart!</a>';
+   return $js_i18n_array;
+};
+```
+
+Following the `update_add_to_cart_message` function, call the `add_filter()` function that is available through the WordPress Plugin API. `add_filter()` hooks a function or method to a specific filter action. Hook `update_add_to_cart_message()` to the `bigcommerce/js_localization` filter hook that is provided by the BC4WP plugin.
+
+```php
+add_filter( 'bigcommerce/js_localization', 'update_add_to_cart_message' );
+```
+
+Once the file is saved, WordPress will now localize the modified string when an item is added to a cart by doing the following:
+
+1. When the `bigcommerce/js_localization` filter hook is applied, `add_filter()` invokes `update_add_to_cart_message()` and `$js_i18n_array` is passed in as an argument.

```

Comment:
1. When you apply the `bigcommerce/js_localization` filter hook, `add_filter()` invokes `update_add_to_cart_message()` and `$js_i18n_array` is passed in as an argument.